app/services/teacher_training_public_api/hydrate_site.rb

```diff
@@ -0,0 +1,35 @@
+module TeacherTrainingPublicAPI
+  class HydrateSite
```

Comment:
could we call this `sync_site` to keep the naming consistent as we use `sync` everywhere else?

Reply:
i did initially call it that, but decided on `hydrate` as the actual syncing is still happening back in `sync_sites` (as i didn't want to throw `@updates`, `@changeset` and `@incremental_sync` all round the codebase. but no major objection to calling it `SyncSite` if you feel it's a better fit.

Reply:
that's fair, I've tried thinking of a way we can surface changes from within this class, which allows us to `save!` the site here and just expose that method but tbh its not a good pattern to follow so happy to keep as is, maybe rename to `AssignSiteAttributes` but happy either way with this or `Hydrate`